### ISSUE TITLE
[Bug]: Appointments are twice in the Calendar (SBZAV)

### ISSUE THREAD

**Issue by MarioHeininger** [mampfes/hacs_waste_collection_schedule#1552]
### I Have A Problem With:

A specific source

### What's Your Problem

With the SBZAV Data i got all appointsments twice. So i have on one days:
- Papiertonnen
- Papiertonnen

on the other Date
- Restmüll
- Restmüll

I checked the ical which comes from SBZAV there are these only single included. I imported that ical to an other calender, looks also fine. 

### Source (if relevant)

sbzav_de

### Logs

```Shell
no relevatn logs
```

### Relevant Configuration

```YAML
waste_collection_schedule:
  sources:
    - name: sbazv_de
      args:
        city: "Ludwigsfelde"
        district: "Ludwigsfelde"
        street: "Sputendorfer+Weg"
```

### Checklist Source Error

- [X] Use the example parameters for your source (often available in the documentation) (don't forget to restart Home Assistant after changing the configuration)
- [X] Checked that the website of your service provider is still working
- [X] Tested my attributes on the service provider website (if possible)
- [X] I have tested with the latest version of the integration (master) (for HACS in the 3 dot menu of the integration click on "Redownload" and choose master as version)

### Checklist Sensor Error

- [X] Checked in the Home Assistant Calendar tab if the event names match the types names (if types argument is used)

### Required

- [X] I have searched past (closed AND opened) issues to see if this bug has already been reported, and it hasn't been.
- [X] I understand that people give their precious time for free, and thus I've done my very best to make this problem as easy as possible to investigate.

**Comment by Cococabana** [comment]
Same issue here!

**Comment by 5ila5** [comment]
duplicate of #1521
fixed by #1522
